server/data/ciagApi/models/updateCiagPlanRequest.ts

```diff
@@ -19,6 +19,7 @@ export default class UpdateCiagPlanRequest {
     this.offenderId = data.offenderId
     this.prisonId = data.prisonId
 
+    this.workOnReleaseReference = data.workOnReleaseReference
     this.desireToWork = data.desireToWork
     this.hopingToGetWork = data.hopingToGetWork
```

Comment:
You can't see it in this diff, but the other objects (e.g. `skillsAndInterests`) are copied directly from the `CiagPlan` in the session. For example:

```
this.workExperience = getValueSafely(data, 'workExperience', null)
```
It's different for these "workOnRelease" values, which have to be mapped individually (as they are at the top level within `CiagPlan`).